docs/apache-airflow/concepts/timetable.rst

```diff
@@ -60,21 +60,44 @@ may be available in plugins.
 CronTriggerTimetable
 ^^^^^^^^^^^^^^^^^^^^
 
-A timetable which accepts a cron expression.
+A timetable that accepts a cron expression, and triggers DAG runs according to it.
 
-Note `CronDataIntervalTimetable`_ also accepts a cron expression. See `Differences between the two cron timetables`_.
+.. seealso:: `Differences between the two cron timetables`_
 
 .. code-block:: python
 
-    from airflow.timetables.interval import CronTriggerTimetable
```

Comment:
This hasn't been released already/it's a new class for 2.4 right?

Reply:
Not yet.